packages/mobile/src/screens/track-screen/TrackScreenDownloadButtons.tsx

```diff
@@ -22,12 +22,14 @@ export type DownloadButtonProps = {
   state: ButtonState
   type: DownloadButtonType
   label: string
+  doesUserHaveAccess: boolean
   onClick?: () => void
 }
 
 export const messages = {
   followToDownload: 'Must follow artist to download',
-  addDownloadPrefix: (label: string) => `Download ${label}`
+  addDownloadPrefix: (label: string) => `Download ${label}`,
+  mustHaveAccess: 'Must have access to download'
```

Comment:
Here also.
This copy is not final. No official copy has been provided for this scenario yet. This should be brought up in the next QA session to make sure it's fine with product/design